ansible_ai_connect/main/settings/base.py

```diff
@@ -171,12 +171,19 @@
 
 SOCIAL_AUTH_LOGIN_ERROR_URL = "/unauthorized/"
 
+
+def is_ssl_enabled(value: str) -> bool:
+    """SSL should be enabled if value is not recognized"""
+    disabled = value.lower() in ("false", "f", "0", "-1")
+    return not disabled
+
+
 AAP_API_URL = os.environ.get("AAP_API_URL")
 AAP_API_PROVIDER_NAME = os.environ.get("AAP_API_PROVIDER_NAME", "Ansible Automation Platform")
-SOCIAL_AUTH_VERIFY_SSL = os.getenv("SOCIAL_AUTH_VERIFY_SSL", "True").lower() in ("true", "1", "t")
-ANSIBLE_AI_MODEL_MESH_API_VERIFY_SSL = os.getenv(
-    "ANSIBLE_AI_MODEL_MESH_API_VERIFY_SSL", "True"
-).lower() in ("true", "1", "t")
+SOCIAL_AUTH_VERIFY_SSL = is_ssl_enabled(os.getenv("SOCIAL_AUTH_VERIFY_SSL", "True"))
+ANSIBLE_AI_MODEL_MESH_API_VERIFY_SSL = is_ssl_enabled(
+    os.getenv("ANSIBLE_AI_MODEL_MESH_API_VERIFY_SSL", "True")
+)
```

Comment:
What about just:
```python
ANSIBLE_AI_MODEL_MESH_API_VERIFY_SSL = not os.getenv(
    "ANSIBLE_AI_MODEL_MESH_API_VERIFY_SSL"
).lower() in ("true", "1", "t")
```

Reply:
The idea of having a function to check if a envvar is Tru-ish makes a lot of sense. But it would be more elegant if it was a generical one, this way we can reuse the function in the rest of the file:

```python
not is_false("ANSIBLE_AI_MODEL_MESH_API_VERIFY_SSL")
```

Reply:
It's harder to test, and a harder to read. Actually if you set up `ANSIBLE_AI_MODEL_MESH_API_VERIFY_SSL` to `True`, you will disable SSL protection with it.

Reply:
And what about the `is_false()` `is_true()` functions, this would allow us to simplify 90% of the configuration file.

Reply:
Sorry @goneri didn't see the second message, had to refresh the page before answering. Yes, `is_false`, `is_true` sounds tempting but adding to the picture default value and now we have `is_false("SOME_VAR", "True")` which already not so elegant. So, yes sounds interesting, but it wasn't the goal for this PR. Let's discuss it offline, how can we unify it. Thank you!